rmgpy/cantherm/statmech.py

```diff
@@ -318,6 +314,19 @@ def load(self):
         logging.debug('    Reading optimized geometry...')
         coordinates, number, mass = geomLog.loadGeometry()
 
+        # Infer atoms from geometry
+        atoms = {}
+        for atom_num in number:
+            try:
+                symbol = atom_num_dict[atom_num]
+            except KeyError:
+                raise Exception('Atom number {} is not yet supported.'.format(atom_num))
```

Comment:
The error message needs a tiny refinement: "atom number" could also be inferred as the number of the atom in the xyz / Z matrix. Could you print the element instead?

Reply:
Yup, I'll change it